org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -80,6 +80,17 @@
                         "package": "rust",
                         "target": "i686-unknown-linux-gnu"
                     }
+                },
+                {
+                    "type": "archive",
+                    "dest": "rust-src",
+                    "url": "https://static.rust-lang.org/dist/2020-11-19/rust-src-1.48.0.tar.xz",
```

Comment:
Update to 1.49.0 and probably rebase, your branch seems missing some latest changes.

Reply:
Both things should be fine now